src/main/java/org/assertj/core/api/AbstractOptionalLongAssert.java

```diff
@@ -0,0 +1,128 @@
+/**
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2015 the original author or authors.
+ */
+package org.assertj.core.api;
+
+import org.assertj.core.internal.Failures;
+import org.assertj.core.internal.Integers;
+import org.assertj.core.util.VisibleForTesting;
+
+import java.util.OptionalLong;
+
+import static org.assertj.core.error.OptionalShouldBeEmpty.shouldBeEmpty;
+import static org.assertj.core.error.OptionalShouldBePresent.shouldBePresent;
+import static org.assertj.core.error.OptionalShouldContain.shouldContain;
+
+/**
+ * Assertions for {@link java.util.OptionalLong}.
+ *
+ * @author Jean-Christophe Gay
+ * @author Alexander Bischof
+ */
+public abstract class AbstractOptionalLongAssert<S extends AbstractOptionalLongAssert<S>> extends
+    AbstractAssert<S, OptionalLong> {
+
+    @VisibleForTesting
+    Integers integers = Integers.instance();
```

Comment:
`Integers` in `AbstractOptionalLongAssert` ??? 
Anyway, we don't need that as per my previous comment.